ansible/ncn_nodes.yml

```diff
@@ -26,10 +26,22 @@
 # NCN (management, storage, worker) as well as their corresponding images.
 # The three seperate top level plays are marked for deprecation via a debug
 # notice.
+---
+- name: setup cfs_image host group
+  gather_facts: false
+  hosts: all
+  tasks:
+    - name: Add image customization hosts to the cfs_image host group.
+      add_host:
+        name: '{{ inventory_hostname }}'
+        groups: cfs_image
+      when: cray_cfs_image
+
+# Does not include images
 - hosts:
-   - Management_Master
-   - Management_Worker
-   - Management_Storage
+   - Management_Master:!cfs_image
```

Comment:
No need to bifurcate the playbook into two for this. We can contextually pass in sysctl set values exactly once in the playbook's vars section via:

      vars:
        sysctl_set: {{ not cray_cfs_image }}